### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by d0990** [KadeArchive/Kade-Engine#17] [Q&A] [answered]
I just want to know how to add weeks, characters, and such

**Comment by noobmanpc** [comment, marked as answer]
@d0990 Changing characters is fairly simple, take the image you would like to replace the current one with and add it to the FNF game "images" folder. Rename the file to the name of the file you want to replace. Example: If you want to replace the sick note with a colored version, rename the colored version to sick. After you do that it should ask if you are sure you want to replace the file with the other one. Just a warning though, you can NOT add the old character back. If you would like to do so, reinstall FNF. Anyway, make sure the animations for your character (yes, you have to create them or else it looks sloppy.) are also replaced. You can do the same with the background, the main menu screen. You can even replace the songs! Not too sure on the mapping, i'm just getting started on that. If you have questions, just reply.
